.github/actions/cleanup-pe/cleantagged.go

```diff
@@ -0,0 +1,170 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/Azure/azure-sdk-for-go/profiles/latest/network/mgmt/network"
+	"github.com/Azure/go-autorest/autorest/azure/auth"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/ec2"
+	"google.golang.org/api/compute/v1"
+
+	"github.com/mongodb/mongodb-atlas-kubernetes/test/e2e/actions/cloud"
+	"github.com/mongodb/mongodb-atlas-kubernetes/test/e2e/api/gcp"
+)
+
+func cleanAllTaggedAWSPE(region, tagName, tagValue string) error {
```

Comment:
All of the "tagged" function versions look a lot like their not-tagged counterparts (close to duplication). What do you think about using the same cleaning function for tagged and non-tagged? You can add a flag or just make a decision based on the fact that tagName and tagValue are both empty.

Reply:
Or make 2 wrapper functions on top of the same core function might be also a great idea